packages/playwright-core/src/server/bidi/bidiNetworkManager.ts

```diff
@@ -50,6 +50,13 @@ export class BidiNetworkManager {
     ];
   }
 
+  async initialize() {
+    const dataTypes: [bidi.Network.DataType] = [bidi.Network.DataType.Response];
+    const maxEncodedDataSize = 20_000_000;
+    const contexts: [string] = [this._session.sessionId];
+    await this._session.send('network.addDataCollector', { dataTypes, maxEncodedDataSize, contexts });
```

Comment:
Is there a reason we do it per page, rather than initialize once per browser session as in https://github.com/microsoft/playwright/pull/37321/files#diff-8753d59ba95abdc3367549f4caf85cbe4fa7b702d4f94427301bc9eadf4e79e1 ?

Reply:
No particular reason, I've updated the PR to do it once per session.